README.md

```diff
@@ -1,17 +1,17 @@
 # ANN
 DBM Fall 2023 Term Project
 
-Collaborators:<br>
-Kevin Nguyen<br>
-David Namenyi<br>
-Kevin Asamoah<br>
-<br>
-Purpose:<br>
-Music application that does various things including playlist management and music trivia<br>
-<br>
-Stack:<br>
-Next + Tailwind CSS<br>
-MySQL<br>
+Collaborators:  
```

Comment:
You can use two spaces at the end of a line instead of a <br> tag. This makes it much more friendly to read in a code editor